demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -86,59 +87,91 @@ mkdir ${BASE_CERT_DIR}
 # cd caikit-tgis-serving/demo/kserve
 
 # Install Service Mesh operators
-oc apply -f custom-manifests/service-mesh/operators.yaml
-echo
-echo "Wait 30s for servicemesh operators"
+echo "[INFO] Install Service Mesh operators"
 echo
-sleep 30
+oc apply -f custom-manifests/service-mesh/operators.yaml
+
+wait_for_csv_installed servicemeshoperator openshift-operators
+wait_for_csv_installed kiali-operator openshift-operators
+wait_for_csv_installed jaeger-operator openshift-operators
 oc wait --for=condition=ready pod -l name=istio-operator -n openshift-operators --timeout=300s
 oc wait --for=condition=ready pod -l name=jaeger-operator -n openshift-operators --timeout=300s
 oc wait --for=condition=ready pod -l name=kiali-operator -n openshift-operators --timeout=300s
 
 # Create an istio instance
-oc create ns istio-system
-sleep 2
-oc apply -f custom-manifests/service-mesh/smcp.yaml
 echo
-echo "Wait 30s for servicemesh control plane"
+echo "[INFO] Create an istio instance"
 echo
-sleep 30
+oc create ns istio-system
```

Comment:
if you use this syntax:
```
oc create ns istio-system -oyaml --dry-run=client | oc apply -f-
```
it allows the script to run happily even if the namespace already exists. Same for `oc create secret`